examples/htool.c

```diff
@@ -1542,6 +1544,105 @@ static const struct htool_cmd CMDS[] = {
                  .desc="The signed cert file."},
                  {}},
     },
+    {
```

Comment:
Can we combine v0 and v1 commands so it's easier to type at the command line? I expect we only support one version or the other at a time, so under the hood we can try both.